### ISSUE TITLE
Under mix mode , when left arrow key is pressed

### ISSUE THREAD

**Issue by muskangupta3** [yairEO/tagify#343]
While working under mix mode , while selecting an option in the dropdown menu , if left arrow key is pressed , the whole input string is erased and only the selected value is shown .
I tried this under the demo also , this could be reproduced there as well .

Thanks in advance

**Comment by yairEO** [comment]
You probably mean "highlighting" a dropdown option and not "selection" it, because that would simply make a tag out of your text and the whole thing you complain about would be invalid.

Anyway, I just tested this and I cannot reproduce... Maybe it's something specific to your computer, that pressing left-arrow does something in the OS-level?

BUT, I did find another bug while testing this one: If you click the right-arrow while highlighting a suggestion from the dropdown, then it autocomplete it **without** the interpolator character at the start, which then will not allow it to become a tag.
